aesara/link/jax/dispatch.py

```diff
@@ -1047,3 +1048,11 @@ def random_variable(rng, size, dtype_num, *args):
         return (rng, smpl_value)
 
     return random_variable
+
+
+@jax_funcify.register(Erf)
+def jax_funcify_Erf(op, node, **kwargs):
```

Comment:
```suggestion
def jax_funcify_Erf(op, **kwargs):
```
Optional, just to standardize with other methods in the module